src/language-js/comments.js

```diff
@@ -980,7 +964,6 @@ module.exports = {
   handleEndOfLineComment,
   handleRemainingComment,
   isTypeCastComment,
-  getGapRegex,
```

Comment:
I believe this only used in js, should we  remove it from comment attach?

Reply:
I'd rather keep it. It's a reasonable extension point. The default regex (`/^[\s(]*$/`) is JS-specific. It might work well for languages that use parens in a way similar to how JS uses them, but it's also possible that other languages might need to override this.